Dockerfile

```diff
@@ -1,67 +1,74 @@
 # syntax=docker/dockerfile:1
 
-FROM node:alpine as build
+######## WebUI frontend ########
+FROM node:21-bookworm-slim as build
```

Comment:
Thanks for the PR! This part shouldn't matter as it's only for the build setp, and is not actually used for deployment. (Check out [multi-stage docker build](https://docs.docker.com/build/building/multi-stage/))

Reply:
Ok, but should we not use fixed versions? Like the node 21 release since pulling the latest without testing could lead to problems in the future. Also using smaller images in a build stage image may not affect the final image size but the build speed. 

But node and bookwork slim have almost the same size.

Reply:
I agree with @jannikstdl about pinning the node version, but I agree with @tjbck that I don't see a need to change from alpine to debian in this stage. It should be of minimal difference one way or another, and usually in those cases I err on the side of what's already in play.

Reply:
Yes as mentioned Debian and alpine are almost exact the same size so we could use 21 apline.  

So we use what's already tested in long term.

Reply:
Ping me when you update!

Reply:
@tjbck ping